client/payment-request/blocks/payment-request-express.js

```diff
@@ -75,8 +75,8 @@ const PaymentRequestExpressComponent = ( {
 		onPaymentRequestAvailable( paymentRequestType );
 	} );
 
-	const onPaymentRequestButtonClick = () => {
-		onButtonClick();
+	const onPaymentRequestButtonClick = ( evt ) => {
+		onButtonClick( evt, paymentRequest );
```

Comment:
```suggestion
	const onPaymentRequestButtonClick = ( event ) => {
		onButtonClick( event, paymentRequest );
```
> [Variable and function names should be full words, using camel case with a lowercase first letter.](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions)

🤷

Reply:
Fair enough. I was matching the variable name used [here](https://github.com/Automattic/woocommerce-payments/blob/4fa44519a2e5da4b723c01e3ba92f513790a4db2/client/payment-request/blocks/use-initialization.js#L85) but happy to change it. 😃

Reply:
Yeah, I know, I didn't like that var there either, but I can't moan at you for that one in this PR. 😂